test/e2e/adminUI/tests/group003List/uxTestListView.js

```diff
@@ -21,184 +21,145 @@ module.exports = {
 
-		browser.adminUIListScreen
-			.click('@firstItemNameValue');
+		browser.adminUIListScreen.clickItemFieldValue([
+			{ row: 1, column: 2, name: 'name', modelTestConfig: NameModelTestConfig, }
+		]);
 
-		browser.adminUIApp
-			.waitForItemScreen();
+		browser.adminUIApp.waitForItemScreen();
 
-		browser.adminUIItemScreen
-			.click('@listBreadcrumb');
+		// TODO: refactor
+		browser.adminUIItemScreen.click('@listBreadcrumb');
 
-		browser.adminUIApp
-			.waitForListScreen();
+		browser.adminUIApp.waitForListScreen();
 	},
 	'List view should allow users to search for items': function (browser) {
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
 
+		// TODO: refactor
 		browser.adminUIListScreen
 			.setValue('@searchInputField', 'Name Field Test 2');
 
-		browser.adminUIApp
-			.waitForListScreen();
+		browser.adminUIApp.waitForListScreen();
 
-		browser.adminUIListScreen
-			.expect.element('@paginationCount').text.to.equal('Showing 1 Name');
+		browser.adminUIListScreen.assertPageItemCountTextEquals('Showing 1 Name');
 
-		browser.adminUIListScreen
-			.expect.element('@firstItemNameValue').text.to.equal('Name Field Test 2');
+		browser.adminUIListScreen.assertItemFieldValueEquals([
+			{ row: 1, column: 2, name: 'name', value: 'Name Field Test 2', modelTestConfig: NameModelTestConfig, },
+		]);
 	},
 	'List view should allow users to clear search filter': function (browser) {
-		browser.adminUIListScreen
-			.click('@searchInputFieldClearIcon');
-
-		browser.adminUIApp
-			.waitForListScreen();
+		browser.adminUIListScreen.clickSearchInputClearIcon();
 
-		browser.adminUIListScreen
-			.expect.element('@paginationCount').text.to.equal('Showing 2 Names');
+		browser.adminUIApp.waitForListScreen();
 
-		browser.adminUIListScreen
-			.expect.element('@firstItemNameValue').text.to.equal('Name Field Test 1');
+		browser.adminUIListScreen.assertPageItemCountTextEquals('Showing 2 Names');
 
-		browser.adminUIListScreen
-			.expect.element('@secondItemNameValue').text.to.equal('Name Field Test 2');
+		browser.adminUIListScreen.assertItemFieldValueEquals([
+			{ row: 1, column: 2, name: 'name', value: 'Name Field Test 1', modelTestConfig: NameModelTestConfig, },
+			{ row: 2, column: 2, name: 'name', value: 'Name Field Test 2', modelTestConfig: NameModelTestConfig, },
+		]);
 	},
 	'List view should allow users to delete items': function (browser) {
-		browser.adminUIListScreen
-			.click('@firstItemDeleteIcon');
+		browser.adminUIListScreen.clickDeleteItemIcon([
+			{ row: 1, column: 1 }
+		])
 
-		browser.adminUIApp
-			.waitForDeleteConfirmationScreen();
+		browser.adminUIApp.waitForDeleteConfirmationScreen();
 
-		browser.adminUIDeleteConfirmation
-			.click('@deleteButton');
+		// TODO: refactor
+		browser.adminUIDeleteConfirmation.click('@deleteButton');
```

Comment:
I have not refactored the Delete Confirmation screen PO, yet.  That would be later!